app/app/Http/Controllers/LocationSearchController.php

```diff
@@ -400,4 +400,16 @@ public function search(Request $request)
 				'search_radius'      => $search_radius
 			]);
 	}
+	/**
+	 * set search radius
+	 */
+	public function setSearchRadius(Request $request) {
+		$distance = Input::get('distance');
+		if(is_numeric($distance)) {
```

Comment:
This is minor but try to consistently follow the spacing that is used in other if-statements.  This applies to all the code you're writing and not just this line.
This is better:
if ( is_numeric($distance) )